src/lib/webworker.generated.d.ts

```diff
@@ -226,9 +230,9 @@ declare var Event: {
 }
 
 interface EventTarget {
-    addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, useCapture?: boolean): void;

```

Comment:
As the library file is generated from the spec file we got from Edge team, this is a reflection of https://github.com/Microsoft/TSJS-lib-generator/blob/master/inputfiles/browser.webidl.xml#L3558